### PR TITLE
DEV-1817 [minor] Add a --version argument to fdt

### PR DESCRIPTION
**Pull request by DrRataplan**
This can be used to upgrade an app to a nightly or staging
version of the sdk without environment variables

### CODE REVIEW

**Comment by mvanbutselaar** [comment]
It would be nice if the --version argument would also be added to the init command.

**Comment by mvanbutselaar** [comment]
Decided to not add this to the init command for now because the use case for this is currently thin.

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Create a new instance of a FontoXML Editor.
 
 Upgrade an instance of a FontoXML Editor, and/or add or remove addons.
 
-	fdt editor run [--cache-control-disabled] [--cache-control-max-age <maxAge>] [--delay <delay>] [--disable-dist-authentication] [--dist] [--lock-not-acquired] [--lock-not-available] [--open] [--port] [--release-locks-on-state-request] [--save-history] [--savemode <savemode>] [--verbose]
+	fdt editor run [--cache-control-disabled] [--cache-control-max-age <maxAge>] [--delay <delay>] [--disable-dist-authentication] [--dist] [--lock-not-acquired] [--lock-not-available] [--open] [--port] [--release-locks-on-state-request] [--save-history] [--savemode <savemode>] [--verbose] [--version <latest|staging>]
```
Comment (evandervalk) [inline README.md]: Should also include 'nightly'.

File: src/api/getSdkBaseUrl.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module.exports = sdkVersion => {
+	switch (sdkVersion) {
+		case 'latest':
+			return process.env.SDK_PORTAL_URL || 'https://sdk.fontoxml.com';
+			break;
+		case 'staging':
```
Comment (evandervalk) [inline src/api/getSdkBaseUrl.js]: There is a chance that the staging environment will be removed leaving only latest and nightly.

File: README.md (hunk 1 above)
Comment (mvanbutselaar) [reply README.md]: As there is talk about not exposing nightly outside of the FontoXML teams, so we opted to not mention it in the README and help output.

File: src/api/getSdkBaseUrl.js (hunk 2 above)
Comment (DrRataplan) [reply src/api/getSdkBaseUrl.js]: If we will remove the staging environment, where will we host out release candidates?

Staging here means "use https://sdk.staging.fontoxml.com".

Comment (mvanbutselaar) [reply src/api/getSdkBaseUrl.js]: Discussed this with Erik. Will discuss further on friday with the correct stakeholders.

File: README.md (hunk 1 above)
Comment (bwrrp) [reply README.md]: Are you sure about that? IIRC nightly is accessible by clients, while staging is strictly for internal testing of RCs. I'd expect the help to mention only latest|nightly.

Comment (DrRataplan) [reply README.md]: @bwrrp, correct: latest, staging and nightly are currently accessible by our partners. The services team chose to hide the presence of a nightly environment from the outside. There is even some talk about only allowing internal access to the staging and nightly portals.

Let's discuss this Friday, when everyone is present.

**Pull request merged**